src/Repository/AccessTokenRepository.php

```diff
@@ -64,6 +67,24 @@ public function findToken(string $token): ?IAccessToken
         return null;
     }
 
+    public function deleteUnusedTokens(): ?int
```

Comment:
Kan deze method niet altijd een int terug geven? Indien de TTL 0 is, gewoon 0 terug geven?

Reply:
Dat is idd een goeie, dan hoef ik ook geen zooi met die sprintf te doen